lib/logger.py

```diff
@@ -0,0 +1,111 @@
+# Pyromania logger ver0
```

Comment:
It'd be great to have a few lines here explaining the logging design. This is the ADTF-like logging, right? With multiple streams in one binary logfile. I am not sure what "Pyromania logger ver0" means.

Reply:
Have I already reviewed this code in the "naio" repository?

Reply:
Yes, this is from "naio" repository. I will add comments into logger.py